views/partials/prs.hbs

```diff
@@ -46,5 +46,9 @@
             {{/each}}
         </div>
     {{/if}}
-{{/exists}}
 
+    <div class="br2 center mt5 overflow-hidden w-50-ns w-80-m w-90 mt2">
+        <button class="bg-orange white center mt2 h2 bn pointer br1 pv0 ph3 db-ns" id="saveUser">This is Me</button>
+        <p class='light-silver center tc-ns'>In the future, you can find your PRs by visiting <a href='/me' class='link white underline-hover'>https://hacktoberfestchecker.herokuapp.com/me</a> on this device.</p>
```

Comment:
Pass the host to the view (https://stackoverflow.com/questions/7507015/get-hostname-of-current-request-in-node-js-express) and use that rather than hard code production url (so it works the same on localhost)

Reply:
Right now the `href` goes to `/me` so it does work on localhost, although I can update the displayed text as well.